packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.ct.tsx

```diff
@@ -93,6 +98,29 @@ test.describe("Empty screenshots", () => {
   });
 });
 
+test.describe("Truncated options screenshots", () => {
+  executeMatrixScreenshotTest({
+    name: "Select (truncated)",
+    columns: DENSITIES,
+    rows: ["ellipsis", "multiline"],
+    disabledAccessibilityRules: DISABLED_ACCESSIBILITY_RULES,
+    component: (column, row) => (
+      <div>
+        <OnyxSelect
+          label="Label"
+          listLabel="List label"
+          options={MOCK_LONG_LABELED_OPTIONS}
+          density={column}
+          truncation={row}
+        />
+      </div>
```

Comment:
the div should be redundant

```suggestion
      <OnyxSelect
        label="Label"
        listLabel="List label"
        options={MOCK_LONG_LABELED_OPTIONS}
        density={column}
        truncation={row}
      />
```